lib/vmpooler.rb

```diff
@@ -46,6 +46,15 @@ def self.config(filepath = 'vmpooler.yaml')
           parsed_config.merge!(extra_config)
         end
       end
+      parsed_config[:config]['pools_from_dir'] = ENV['POOLS_FROM_DIR'] if ENV['POOLS_FROM_DIR']
+      if parsed_config[:config]['pools_from_dir'] && Dir.exist?(parsed_config[:config]['pools_from_dir'])
+        pools_from_dir = []
+        Dir["#{parsed_config[:config]['pools_from_dir']}/*.yaml"].sort.each do |file_name|
+          parsed_pool = Yaml.load_file(file_name)
+          pools_from_dir += parsed_pool
```

Comment:
turns out load_file returns `false` if the file exists but fails to parse as yaml. since false in your array parsed_pool would not work i suggest
```suggestion
          pools_from_dir += parsed_pool if parsed_pool
```

Reply:
Good call. Adding